parent/pom.xml

```diff
@@ -493,6 +493,20 @@
             <version>${version.hibernate.entitymanager}</version>
          </dependency>
          <dependency>
+            <groupId>org.jboss.shrinkwrap.resolver</groupId>
+            <artifactId>shrinkwrap-resolver-bom</artifactId>
+            <version>${version.shrinkwrapResolver}</version>
+            <type>pom</type>
+            <scope>import</scope>
+         </dependency>
+         <!-- This is a work around for https://issues.jboss.org/browse/SHRINKRES-238 -->
+         <!-- After fixing the issue above, the dependency below can be safely removed -->
```

Comment:
By Uber Jars I assume you mean `shrinkwrap-resolver-bom`...

It is the recommended way of using the resolvers: https://github.com/shrinkwrap/resolver
> Obtaining ShrinkWrap Resolvers for use in your system can be done in a single pass by declaring a dependency upon the depchain module in a Maven pom.xml:
```
<dependencies>
    ...
    <dependency>
      <groupId>org.jboss.shrinkwrap.resolver</groupId>
      <artifactId>shrinkwrap-resolver-depchain</artifactId>
      <version>${version.shrinkwrap.resolvers}</version>
      <scope>test</scope>
      <type>pom</type>
    </dependency>
    ...
</dependencies>
```

However it seems you are correct - `shrinkwrap-resolver-impl-maven` can be removed. However I still couldn't get rid of `maven-aether-provider`.